GitFyle.Core.Api/Services/Foundations/Configurations/ConfigurationService.Exceptions.cs

```diff
@@ -16,6 +17,33 @@ namespace GitFyle.Core.Api.Services.Foundations.Configurations
     internal partial class ConfigurationService
     {
         private delegate ValueTask<Configuration> ReturningConfigurationFunction();
+        private delegate ValueTask<IQueryable<Configuration>> ReturningConfigurationsFunction();
+
+        private async ValueTask<IQueryable<Configuration>> TryCatch(ReturningConfigurationsFunction returningConfigurationsFunction)
```

Comment:
```suggestion
        private async ValueTask<IQueryable<Configuration>> TryCatch(
            ReturningConfigurationsFunction returningConfigurationsFunction)
```
This line is over 120.

Reply:
I have this issue. Thanks.